src/model/groups.rs

```diff
@@ -114,3 +116,78 @@ pub struct GroupAuditLogDataChange<T> {
 	/// The new field after the change.
 	pub new: T,
 }
+
+#[derive(Clone, Debug, Deserialize, Eq, PartialEq, Serialize)]
+#[serde(rename_all = "camelCase")]
+/// Details about a group un-ban.
+pub struct GroupBan {
+	/// Unique identifier of the un-ban.
+	pub id: String,
+	/// Identifier of the group.
+	pub group_id: crate::id::Group,
+	/// Identifier of the user who was un-banned.
+	pub user_id: crate::id::User,
+	/// Flag indicating if the user was representing the group at the time of
+	/// un-ban.
+	pub is_representing: bool,
```

Comment:
Ban/Unban? Or if both, I'd make the docs just state that a bit more clearly with ban/un-ban, or the like...

Reply:
Oh yeah, it originally wrote (un)ban but that's a spelling mistake, so I replaced it with un-ban without re-reading the code.